src/i18n/locales/ca.json

```diff
@@ -11,7 +11,7 @@
     "votes_one": "<span>1</span> <text>votant</text>",
     "votes_many": "<span>{{ count }}</span> <text>votants</text>",
     "votes_other": "<span>{{ count }}</span> <text>votants</text>",
-    "voting_anonymous_advice": "Si us plau, espera mentre la màgia té lloc.Això pot trigar uns minuts; no tanquis la finestra."
+    "voting_anonymous_advice": "Si us plau, espera mentre la màgia té lloc \uD83E\uDE84.\nAixò pot trigar uns minuts; no tanquis la finestra."
```

Comment:
Why not simply use the emoji 🪄?

Reply:
When I copied the emoji to the json the IDE just pasted the hex, don't know why... I just pushed a fix 566632e